crates/torii/core/src/sql.rs

```diff
@@ -67,14 +68,16 @@ impl Sql {
 
     pub async fn head(&self) -> Result<(u64, Option<Felt>)> {
         let mut conn: PoolConnection<Sqlite> = self.pool.acquire().await?;
-        let indexer_query = sqlx::query_as::<_, (i64, Option<String>)>(
-            "SELECT head, pending_block_tx FROM indexers WHERE id = ?",
+        let indexer_query = sqlx::query_as::<_, (Option<i64>, Option<String>, String)>(
+            "SELECT head, pending_block_tx, contract_type FROM contracts WHERE id = ?",
         )
         .bind(format!("{:#x}", self.world_address));
 
-        let indexer: (i64, Option<String>) = indexer_query.fetch_one(&mut *conn).await?;
+        let indexer: (Option<i64>, Option<String>, String) =
+            indexer_query.fetch_one(&mut *conn).await?;
+        assert!(indexer.2 == WORLD_CONTRACT_TYPE);
```

Comment:
I think we should avoid using asserts except in tests

Reply:
Yep should be removed here.
In some cases even not in test I would say it can definitely put some nets.
But in the current case, this may be false by design.